AsyncFixer.Test/AsyncVoidTests.cs

```diff
@@ -63,6 +63,32 @@ private static async Task foo()
             VerifyCSharpFix(test, fixtest);
         }
 
+        [Fact]
+        public void AsyncVoidAnonymousFunction()
+        {
+            var test = @"
+using System;
+using System.Threading.Tasks;
+
+class Program
+{
+    private static void foo()
+    {
+        bar(async i => await Task.FromResult(i));

```

Comment:
Is it possible that the indirection by using another method too here like `ProcessAsync()` in the [original sample](https://github.com/semihokur/AsyncFixer/issues/2) would throw it off?

Reply:
Added another test case for that.

Reply:
Thank you!